velox/exec/RowsStreamingWindowBuild.cpp

```diff
@@ -28,6 +40,7 @@ RowsStreamingWindowBuild::RowsStreamingWindowBuild(
   velox::common::testutil::TestValue::adjust(
       "facebook::velox::exec::RowsStreamingWindowBuild::RowsStreamingWindowBuild",
       this);
+  rangeFrame_ = hasRangeFrame(windowNode);
```

Comment:
This could be a const and set in iniializer list.
s/rangeFrame_/hasRangeFrame_/